component_metadata.yml

```diff
@@ -0,0 +1,4 @@
+dependencies:
+- constraints:
+  - version<2
```

Comment:
Why the cap?  Upstream openstack doesn't cap deps like this without good reason, we shouldn't either.

Reply:
@prometheanfire  That constraint can be removed -- constraint specification is entirely down to the project's team. I just want to show how to get this tooling integrated.

Reply:
I'd prefer it to be removed then :P